core/templates/dev/head/pages/get_started/get_started.html

```diff
@@ -1,102 +1,110 @@
+            </p>
+
+            <h2 translate="I18N_GET_STARTED_PAGE_PARAGRAPH_2_HEADING"></h2>
+            <p translate="I18N_GET_STARTED_PAGE_PARAGRAPH_2">
+            </p>
+
+            <h2 translate="I18N_GET_STARTED_PAGE_PARAGRAPH_3_HEADING"></h2>
+            <p translate="I18N_GET_STARTED_PAGE_PARAGRAPH_3">
+            </p>
+            <p translate="I18N_GET_STARTED_PAGE_PARAGRAPH_4">
+
+            </p>
+            <p translate="I18N_GET_STARTED_PAGE_PARAGRAPH_5">
+
+            </p>
+            <ul style="margin-left: 40px;">
+              <li><a href="https://www.oppia.org/explore/0" target="new">Welcome to Oppia</a></li>
+              <li><a href="https://www.oppia.org/collection/wqCTKpKA0LBe" target="new">Get Electrified!</a></li>
+              <li><a href="https://www.oppia.org/collection/inDXV0w8-p1C" target="new">Programming with Carla</a></li>
+            </ul>
+            <p translate="I18N_GET_STARTED_PAGE_PARAGRAPH_6">
+            </p>
+
+            <h2 translate="I18N_GET_STARTED_PAGE_PARAGRAPH_7_HEADING"></h2>
+            <p translate="I18N_GET_STARTED_PAGE_PARAGRAPH_7">
+            </p>
+
+            <h2 translate="I18N_GET_STARTED_PAGE_PARAGRAPH_8_HEADING"></h2>
+            <p translate="I18N_GET_STARTED_PAGE_PARAGRAPH_8">
+            </p>
+
+            <!-- todo: make it easy to get the right link to share, update this doc with that info -->
+
+            <h2 translate="I18N_GET_STARTED_PAGE_PARAGRAPH_9_HEADING"></h2>
+            <p translate="I18N_GET_STARTED_PAGE_PARAGRAPH_9">
+            </p>
+            <p translate="I18N_GET_STARTED_PAGE_PARAGRAPH_10">
+
+            </p>
+
+            <h2 translate="I18N_GET_STARTED_PAGE_PARAGRAPH_11_HEADING"></h2>
+            <p translate="I18N_GET_STARTED_PAGE_PARAGRAPH_11">
+            </p>
+          </div>
+        </div>
+        <div class="oppia-footer-padding-below-banner">
+        </div>
+      </content>
+
+      <navbar-breadcrumb>
+        <ul class="nav navbar-nav oppia-navbar-breadcrumb">
+          <li>
+            <span class="oppia-navbar-breadcrumb-separator"></span>
+            Get Started
+          </li>
+        </ul>
+      </navbar-breadcrumb>
+    </base-content>
+
+    <style>
+      .oppia-get-started h2 {
+        font-size: 1.17em;
+        font-weight: bold;
+        text-align: left;
+      }
+    </style>
+
+    <script src="/templates/dev/head/AppInit.js"></script>
+    <!-- This code is used for inserting webpack bundles
+        https://github.com/jantimon/html-webpack-plugin#writing-your-own-templates -->
     <% if (webpackConfig.mode == 'production') { %>
-      <script src="/build/templates/head/dist/<%= htmlWebpackPlugin.files.js[chunk] %>"></script>
+      <script src="/third_party/generated/js/third_party.min.js"></script>
     <% } else { %>
-      <script src="/dist/<%= htmlWebpackPlugin.files.js[chunk] %>"></script>
+      <script src="/third_party/generated/js/third_party.js"></script>
```

Comment:
Please use `@require('../../footer_js_libs.html')` here and modify _footer_js_libs.html_ to use webpack.

Reply:
Yes, that makes sense

Reply:
Done